offset_manager_test.go

```diff
@@ -105,6 +107,111 @@ func TestNewOffsetManager(t *testing.T) {
 	}
 }
 
+// Test that the correct sequence of offset commit messages is sent to a broker when
+// multiple goroutines for a group are committing offsets at the same time
+func TestOffsetManagerCommitSequence(t *testing.T) {
+	mu := &sync.Mutex{}
```

Comment:
Same as below, just go ahead and declare the variable instead: `var mu sync.Mutex`

Reply:
Superseded by switch to sync/atomic.Pointer - which I've declared as per your suggestion.